src/main/java/seedu/trippie/command/LoadTripCommand.java

```diff
@@ -20,6 +20,9 @@ public void execute(Ui ui, TrippieData trippieData) {
             try {
                 System.out.print("Which one do you want to load? Enter the index:");
                 index = Integer.parseInt(ui.getLine());
+                if (index < 0) {
+                    throw new IndexOutOfBoundsException("Please enter a valid trip number");
+                }
```

Comment:
Already resolved in Issue #82, don't need to add as IndexOutOfBoundsException will already be thrown when loadCurrentTripFromFile() cannot access the index.

Reply:
Roger